### PR TITLE
add sleepIn

### PR DESCRIPTION
**Pull request by WarrenKnocker**

### CODE REVIEW

File: WarmUp1/sleepIn/src/index.ts [hunk 1]
```diff
@@ -0,0 +1,34 @@
...
+public boolean sleepIn(boolean weekday, boolean vacation)
+*/
+
+export function sleepIn(weekday: boolean, vacation: boolean): boolean {
+
+if (weekday == false && vacation == false ) {
+    return true;
+}
+
+else if (weekday == false && vacation == false ) {
```
Comment (JustinGuedes) [inline WarmUp1/sleepIn/src/index.ts]: What's the difference between the above condition?

File: WarmUp1/sleepIn/src/index.ts [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+
+sleepIn(false, false) → true
+sleepIn(true, false) → false
+sleepIn(false, true) → true
+
+Java:
+public boolean sleepIn(boolean weekday, boolean vacation)
+*/
+
+export function sleepIn(weekday: boolean, vacation: boolean): boolean {
```
Comment (JustinGuedes) [inline WarmUp1/sleepIn/src/index.ts:16]: Please format the function

Comment (JustinGuedes) [inline WarmUp1/sleepIn/src/index.ts:16]: Also please simplify the function

**Review by JustinGuedes — CHANGES_REQUESTED** [review]

**Comment by WarrenKnocker** [comment]
Request to merge. Simplified function

**Pull request merged**